lib/rubocop/cop/rspec/let_setup.rb

```diff
@@ -3,7 +3,7 @@
 module RuboCop
   module Cop
     module RSpec
-      # Detect unreferenced `let!` calls being used for test setup
+      # Checks for `let!` being used for test setup.
```

Comment:
Updated. FYI the main thing I was going for was trying to standardize messages using "Check" or "checks" when possible. You are right though